src/App.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { ChangeEvent, FormEvent } from "react";
+import React, { type ReactElement, type ChangeEvent, type FormEvent } from 'react';
```

Comment:
Why did you add `type` for type imports?

Reply:
It was suggested by eslint for this Rule: https://typescript-eslint.io/rules/consistent-type-imports

Reply:
I see, nothing wrong here just unusual to me.